### ISSUE TITLE
Split data and train classifier

### ISSUE THREAD

**Issue by zietzm** [zietzm/NNquant#3]
I had been thinking of a 60/20/20 split train/validation/test would be reasonable.

It seems that a useful first step for classifiers would be the following: gain (~ >0.5 % / day?), no change (-0.5% < change < +0.5%), and loss (~ < -0.5 % / day). We should attempt to classify stocks based on the extracted features into one of the categories. 

The validation data set could be very useful for adjusting the hyperparameter, on the threshold beyond which we classify daily change as a gain or loss.

Splitting data could use `sklearn.cross_validation.train_test_split` or `sklearn.model_selection.train_test_split` , potentially?
